### ISSUE TITLE
Support email address with plus addressing as username

### ISSUE THREAD

**Issue by martinvandiemen** [BIG-RAT/jamfcpr#14]
Please add support for email addresses with plus addressing like test+sample@domain.tld.

Currenty jamfcpr reports that the credentials are invalid.

**Comment by BIG-RAT** [comment]
The account works fine generating a token using the web console?

**Comment by martinvandiemen** [comment]
Yes. Btw. Prune and Object Info work with and email address containing a plus.

**Comment by BIG-RAT** [comment]
Thanks for the added info.  It’s odd as they should all be using the same authentication method.  I’ll have to do some digging.
